### DISCUSSION TITLE
Fermata can go full screen?

### DISCUSSION THREAD

**Discussion by loveyoulovemetn** [AndreyPavlenko/Fermata#483] [General]
![IMG_9177](https://github.com/AndreyPavlenko/Fermata/assets/168961589/16d24c8a-73df-4ec9-91f9-dbd9c4073e1f)
Vinfast VF6, MHU android 9
Phone Pixel 4, Android 13
Fermata cant go full screen
Youtube music cant go full screen also
Google maps, Navitel can go full screen
How can we make Fermata full screen as google maps? Many thanks

![IMG_9179](https://github.com/AndreyPavlenko/Fermata/assets/168961589/de48720e-64b5-46e0-afde-1ec45b6b050e)

**Comment by AndreyPavlenko** [comment]
Only the navigation apps support full screen. You may use the screen mirroring with Fermata Mirror FS.

**Reply by loveyoulovemetn** [reply]
Fermata on my old car is still able to go full screen on  the same pixel 4
So this depends on android auto version by car not by phone?

**Comment by AndreyPavlenko** [comment]
It depends on the screen size. On Small screens AA does not use the split screen.

**Reply by loveyoulovemetn** [reply]
Old car = 15.6 inch screen fermata can go full screen
New car = 12.9 inch fermata cant go full screen
Is that reasonable?

**Reply by AndreyPavlenko** [reply]
I don't know what exact logic is implemented in AA. The app does not have control over it. You could make experiments with [DHU](https://developer.android.com/training/cars/testing/dhu) by changing the screen resolution.

**Comment by voxxx70** [comment]
I have infotainment in my car in resolution 1280x680. How to change in full screen in IPTV app? When I put in Fill screen I dont see all picture. The pic is stratched. Is this possible to fix it?

**Reply by AndreyPavlenko** [reply]
Use 'Best fit'. It will scale full screen without cropping, but if the picture resolution is not proportional to the screen size, you'll have black horizontal or vertical paddings.

**Reply by voxxx70** [reply]
Then this is problem with resolution of IPTV channels and resolution of my media. Ok. What is the best of Fermata version for Android 9? Do you plan to add Netflix in Fermata auto?

**Reply by AndreyPavlenko** [reply]
Try this beta version - https://app.box.com/s/vafpy7bhjwnsev9j8hnryv735irrtaj8. For A9 you need KingInstaller.
I've not tried, but probably you can watch Netflix using the screen mirroring.
